src/components/TypeComponents/SimpleSearch/Widget/simpleSearchPathBuilder.js

```diff
@@ -84,16 +84,20 @@ const simpleSearchPathBuilder = (widgetDef, widgetConf, stripes) => {
 
       // Then take each of the rules within the filter, and OR them together with the correct comparators
       const { rules } = f;
+
       rules.forEach((r, ind) => {
-        if (r.comparator === 'isNull' || r.comparator === 'isNotNull') {
+        if (r.comparator === 'isNull' 
+        || r.comparator === 'isNotNull' 
```

Comment:
Agai here as well please, you can do:
```
if (
  condition A ||
  condition B ||
  condition C
)
```
it just makes it a bit easier to parse later

Reply:
sure, will do